src/main/java/com/revature/models/CurriculumTopic.java

```diff
@@ -42,23 +42,23 @@
 	
 //	@NotNull
 	@Column(name="number_of_weeks")
-	private Integer number_of_weeks;
+	private Integer numberOfWeeks;
 	
 	public CurriculumTopic() {}
 
 	public CurriculumTopic(Integer curriculum_id, Integer topic_id, Integer number_of_weeks) {
```

Comment:
Why are curriculum id, topic id and number of weeks still snake cased? are they snake cased somewhere else within the application?